hw/arm/t8030.c

```diff
@@ -105,7 +105,7 @@
 #define T8030_SIO_DATA_SIZE     (0xf8000)
 #define T8030_SIO_DATA_REMAP    (0x220000)
 
-#define T8030_PANIC_BASE        (0x8ffeb0000)
+#define T8030_PANIC_BASE        (0x8fc2b4000)
```

Comment:
Why?

Reply:
To accomodate the risen vram size, so it won't overlap.

Reply:
*accommodate